rubato/utils/vector.py

```diff
@@ -526,10 +528,12 @@ def __add__(self, other: any) -> Vector:
     __radd__ = __add__
 
     def __sub__(self, other: any) -> Vector:
-        if isinstance(other, (int, float)):
-            return Vector(self.x - other, self.y - other)
         if isinstance(other, Vector):
             return Vector(self.x - other.x, self.y - other.y)
+        if isinstance(other, (int, float)):
+            return Vector(self.x - other, self.y - other)
+        if isinstance(other, (tuple, list)):
+            return Vector(self.x - other[0], self.y - other[1])
 
     def __rsub__(self, other: any) -> Vector:
         return Vector(other - self.x, other - self.y)
```

Comment:
only half implemented tuple support.

Reply:
Also make sure to add docstrings for all not private functions

Reply:
done